tests/test_models.py

```diff
@@ -138,3 +142,51 @@ def test_build(simai_client):
     model_conf.pop("project_id")
 
     assert model_conf == MODEL_CONF_RAW
+
+
+def test_set_doa(simai_client):
+    """WHEN the Domain of Analysis is updated
+    THEN the simulation_volume is updated accordingly
+    """
+
+    model_conf = ModelConfiguration(project_id=MODEL_RAW["project_id"], **MODEL_CONF_RAW)
+
+    doa = model_conf.domain_of_analysis
+
+    new_height = {"position": "relative_to_center", "value": 0.5, "length": 15.2}
+
+    doa.Height = simai_client.models.doa_axis_definition(**new_height)
+
+    model_conf.domain_of_analysis = doa
```

Comment:
When I set `model_conf.domain_of_analysis.height = DomainAxisDefinition(...)` I expect `model_conf` to be updated !
This to me really needs to be solved.